kge/model/transd.yaml

```diff
@@ -0,0 +1,13 @@
+import: [lookup_embedder]
+
+transd:
+  class_name: TransD
+  entity_embedder:
+    type: lookup_embedder
+    +++: +++
+  relation_embedder:
+    type: lookup_embedder
+    +++: +++
+
+  l_norm: 1.
```

Comment:
In the paper they always use l_norm = 2, would it make sense to set that as the default value?

Reply:
I think it makes more sense to use l_norm = 2. But the default value of the other models such as TransE, TransH and RotatE is l_norm = 1. So I just didn't touch it.